src/bot/commands/lastfm/fm.ts

```diff
@@ -153,8 +155,101 @@ export class LastFm extends Command {
                 }
                 break;
             case 'artist':
```

Comment:
This is huge for a single switch case. Consider putting it into a function (or several).